snippets/card-product.liquid

```diff
@@ -216,7 +217,7 @@
     </div>
   </div>
 {%- else -%}
-  <div class="product-card-wrapper card-wrapper underline-links-hover">
+  <div class="card-wrapper underline-links-hover {% if wrapper_class == '' %}product-card-wrapper{% else %}{{ wrapper_class }}{% endif %}">
```

Comment:
Follow up to the comment above: here, we are outputting the dynamic `wrapper_class` value that matches `{{ section.settings.card_styles }}`.

So for these cards, it's inconsistent with the `image` and `video` ones. I would suggest to either use the future-proof `{{ section.settings.card_styles }}` to render classes, or hardcode `product-card-wrapper` if we think that's the only value we'll have.

Reply:
Since the card product will always be styled as card product with whichever setting is picked (at least for now), I have stripped the logic from here since I realized it's redundant